apps/condo/domains/ticket/schema/TicketChange.test.js

```diff
@@ -84,93 +83,91 @@ describe('TicketChange', () => {
 
             const [updatedTicket] = await updateTestTicket(admin, ticket.id, payload)
 
-            const objs = await TicketChange.getAll(admin, {
-                ticket: { id: ticket.id },
-            })
-
-            expect(objs).toHaveLength(1)
-            expect(objs[0].id).toBeDefined()
-            expect(objs[0].v).toEqual(1)
-            expect(objs[0].dv).toEqual(1)
-            expect(objs[0].sender).toEqual(updatedTicket.sender)
-            expect(objs[0].detailsFrom).toEqual(ticket.details)
-            expect(objs[0].detailsTo).toEqual(payload.details)
-            expect(objs[0].numberFrom).toEqual(ticket.number)
-            expect(objs[0].numberTo).toEqual(payload.number)
-            expect(objs[0].statusReasonFrom).toEqual(ticket.statusReason)
-            expect(objs[0].statusReasonTo).toEqual(payload.statusReason)
-            expect(objs[0].clientNameFrom).toEqual(ticket.clientName)
-            expect(objs[0].clientNameTo).toEqual(payload.clientName)
-            expect(objs[0].clientEmailFrom).toEqual(ticket.clientEmail)
-            expect(objs[0].clientEmailTo).toEqual(payload.clientEmail)
-            expect(objs[0].clientPhoneFrom).toEqual(ticket.clientPhone)
-            expect(objs[0].clientPhoneTo).toEqual(payload.clientPhone)
-            expect(objs[0].isEmergencyFrom).toEqual(ticket.isEmergency)
-            expect(objs[0].isEmergencyTo).toEqual(payload.isEmergency)
-            expect(objs[0].isWarrantyFrom).toEqual(ticket.isWarranty)
-            expect(objs[0].isWarrantyTo).toEqual(payload.isWarranty)
-            expect(objs[0].isPaidFrom).toEqual(ticket.isPaid)
-            expect(objs[0].isPaidTo).toEqual(payload.isPaid)
-            expect(objs[0].sectionNameFrom).toEqual(ticket.sectionName)
-            expect(objs[0].sectionNameTo).toEqual(payload.sectionName)
-            expect(objs[0].floorNameFrom).toEqual(ticket.floorName)
```

Comment:
Why you have replaced all these strings from `objs[0]` to `ticket1`?
Since we have only one subject instance, it's simpler to have an abstract variable name like `obj`/`objs` and not to use domain-specific names.

Reply:
I've replaced `getAll` by `getOne` since we are always receiving single object here by id, and it contains ticket entity, so I named it correspondingly.